runtime/doc/api.txt

```diff
@@ -3336,6 +3336,16 @@ nvim_tabpage_set_var({tabpage}, {name}, {value})
       • {name}     Variable name
       • {value}    Variable value
 
+nvim_tabpage_set_win({tabpage}, {win})                *nvim_tabpage_set_win()*
+    Sets the current window in a tabpage
```

Comment:
I'd mention explicitly somehow that `{win}` already needs to be in the tabpage (it is obvious to anyone already familiar with the "model", but it is good if the docs are friendly even to someone just getting used to it).

Reply:
Ahh yes that's a good point, thanks!